content/en/blog/_posts/2020-08-21-Moving-Forward-From-Beta/index.md

```diff
@@ -0,0 +1,111 @@
+features from beta to GA than from alpha to beta. Vendors who want a particular feature
+have had good reason to help get code to the point where features are enabled by default,
+and beyond that the journey has been less clear.
+
+KEPs track more than code improvements. Essentially, anything that would need
+communicating to the wider community merits a KEP. That said, most KEPs cover
+Kubernetes features (and the code to implement them).
+
+You might know that [Ingress](/docs/concepts/services-networking/ingress/)
+has been in Kubernetes for a while, but did you realize that it actually went beta in 2015? To help
+drive things forward, Kubernetes' Architecture Special Interest Group (SIG) have a new approach in
+mind.
+
+## Avoiding permanent beta
+
+For Kubernetes REST APIs, when a new feature's API reaches beta, that starts a countdown.
+The beta-quality API now has **nine calendar months** to either:
+- reach GA, and deprecate the beta, or
+- have a new beta version (_and deprecate the previous beta_).
+
+To be clear, at this point **only REST APIs are affected**. For example, _APIListChunking_ is
+a beta feature but isn't itself a REST API. Right now there are no plans to automatically
+deprecate _APIListChunking_ nor any other features that aren't REST APIs.
+
+If a REST API reaches the end of that 9 month countdown, then the next Kubernetes release
+will deprecate that API version. There's no option for the REST API to stay at the same
+beta version beyond the first Kubernetes release to come out after the 9 month window.
+
+### What this means for you
+
+If you're using Kubernetes, there's a good chance that you're using a beta feature. Like
+I said, there are lots of them about.
+As well as Ingress, you might be using [CronJob](/docs/concepts/workloads/controllers/cron-jobs/),
+or [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/), or others.
+There's an even bigger chance that you're running on a control plane with at least one beta
+feature enabled.
+
+If you're using or generating Kubernetes manifests that use beta APIs like Ingress, you'll
+need to plan revise those. The current APIs are going to be deprecated following a
+schedule (the 9 months I mentioned earlier) and after a further 9 months those deprecated
+APIs will be removed. At that point, to stay current with Kubernetes, you should already
+have migrated.
+
+### What this means for Kubernetes contributors
+
+The motivation here seems pretty clear: get features stable. Guaranteeing that beta
+features will be deprecated adds a pretty big incentive so that people who want the
+feature continue their effort until the code, documentation and tests are ready for this
+feature to graduate to stable, backed by several Kubernetes' releases of evidence in
+real-world use.
+
+### What this means for the ecosystem
+
+In my opinion, these harsh-seeming measures make a lot of sense, and are going to be
```

Comment:
This is great @sftim! A minor nit, I would drop "In my opinion," since the post is voiced as you and it doesn't add or remove value.

```suggestion
These harsh-seeming measures make a lot of sense, and are going to be
```

(non blocking feedback)

Reply:
So as not to block publishing, I think I'd prefer to get this approved and _then_ tweak the wording.

Reply:
I don't think publishing is blocked. There seems to be time to get in a few corrections?